src/rules/selector-descendant-combinator-no-non-space/README.md

```diff
@@ -0,0 +1,30 @@
+# selector-descendant-combinator-no-non-space
+
+Disable non-spaces characters of the descendant combinator in descendant selectors.
+
+```css
+   .selector1 .selector2 .selector3 { color: pink; }
```

Comment:
Can you update these (and the patterns below) to `.foo`, `.bar` and `.baz` for consistency with the other READMEs?